_events/2022/2022-03-dei-speaker-series.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Allies, Collaboration & Automation - Working towards inclusive Engagement in Research Computing
+subtitle: US-RSE DEI-WG Speaker Series
+expires: 2021-03-11
```

Comment:
Still this question of if to expire the day after.

Reply:
yep I'd recommend that. If the site doesn't get changes on that day it would be fine, but if the site is built on that day it will stop showing up.

Reply:
```suggestion
expires: 2021-03-12
```

Reply:
I think this final suggestion should be applied before merge.